volatility3/framework/symbols/linux/__init__.py

```diff
@@ -345,6 +345,24 @@ def files_descriptors_for_process(
 
                 yield fd_num, filp, full_path
 
+    @classmethod
+    @Deprecation.deprecated_method(
+        replacement=linux_utilities_modules.Modules.mask_mods_list,
+        replacement_base_class_required_version=(1, 0, 0),
```

Comment:
I'd probably just go with `replacement_version`, I'm all for descriptive names, but that's getting a little excessive, even if we'll only use them a few times.  If they need to know what the version is for, the decorator should document that.